src/content/docs/style-guide/writing-docs/writer-workflow/process-changes.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Proposing process or workflow changes
+topics:
+  - Tech writer style guide
+  - Processes and procedures
+---
+
+Use this document to learn how to propose changes to the process or workflow in a collaborative manner.
+
+## Proposing a change
+
+Do you have an idea you want to try out? It can be something big or something little or even something you haven't figured out how to solve yet. It can be a small suggestion such as using conventional commits, or a large scale change such as automating PR reviews. If so, follow these steps so the team can collaborate on the issue or suggestion.
+
+1. Write your idea as a GitHub issue. The issue should have a brief overview, the actual proposal, and any relevant action items. See [this issue](https://github.com/newrelic/docs-website/issues/4553) for an example. If all you have is an issue and no proposal, that is fine too. The team can collaborate to solve it in the next step.
+2. Once you've done that, let other people know about it in Slack. Try to invite both positive and negative feedback. Not everyone may love a suggestion in it's first iteration. Avoid team-wide mentions unless it is a change that will drastically affect the entire team.
+3. Feedback and conversations over the suggestion should happen in the issue itself. This will reduce Slack fatigue and keep all the information organized in one place. 
+4. Once you've got feedback, finalize the proposal in the form of a pull request against our style guide or project management handbook. 
+5. Let the team know you've submitted the final proposal in either Slack or a team meeting. The team has 7 days to continue collaborating on it or dig their heels in. If there are still conversations ongoing after 7 days, try to resolve those and figure out why there is friction. 
+6. Once the 7 days are over or there is a team-wide concensus, merge the pull request in. It is now a formal team process!
```

Comment:
```suggestion
6. Once the 7 days are over or there is a team-wide consensus, merge the pull request in. It's now a formal team process!
```

Reply:
Maybe instead of team-wide consensus, we should use a consent model. It flips things a little. Instead of getting everyone to explicitly buy in to a thing before moving forward, we move forward unless there are strong objections. This lets us move forward in the face of silence or low engagement. Makes it easier to try things out and change things later if it doesn't work.

Reply:
same comment re: spelling out numbers